### ISSUE TITLE
New Function (New-MrkNetworkClient)

### ISSUE THREAD

**Issue by CalebBartle** [sanderkl/PSMeraki#12]
Here's a new Function I made that uses PSMeraki as the backbone. Enjoy.
function New-MrkNetworkClient {
    [CmdletBinding()]
    Param (
        [Parameter(Mandatory)][ValidateNotNullOrEmpty()][String]$networkId,
        [Parameter(Mandatory)][ValidateNotNullOrEmpty()][String]$clientMac,
        [Parameter(Mandatory)][ValidateNotNullOrEmpty()][String]$clientName,
        [Parameter()][String]$devicePolicy
    )
    $body = @{
        "clients" =@(    
            ([PSCustomObject]$clientbody =@{
                "mac" = $clientMac
                "name" = $clientName
            })
        )
        "devicePolicy" = $devicePolicy
     }
        $request = Invoke-MrkRestMethod -Method POST -ResourceID ('/networks/' + $networkId + '/clients/provision') -Body $body
}

**Comment by sanderkl** [comment]
Hi Caleb,

Sorry for my late response, but I found  your feedback. thanks! 
Its great to hear that this module is being used. 

I added the new function to the module, but I gave it a different name. 
Looking back, I should have named the New-MrkDevice function something that included the word "claim" since that's what it does. 
this new function "provisions" a device. so I added it under the name: 
New-MrkDeviceProvision

I see the spec states that multiple clients can be provisioned in one go (the property is plural, "clients") while current implementation only allows to provision one device at the time. That is some room for improvement.

Unfortunately I lost access to Meraki hardware (i got a new customer who does not use Meraki) so I cannot test it. 

Cheers!
